src/main/kotlin/calculator/StringAddCalculator.kt

```diff
@@ -49,18 +35,14 @@ class StringAddCalculator {
         }
     }
 
-    fun stringAdd(input: List<String>): Int {
-        return input.sumOf { it.toInt() }
-    }
-
     private fun containsDelimiter(input: String): Boolean {
         return input.contains(DELIMITER_COMMA) || input.contains(DELIMITER_COLON)
     }
 
     companion object {
+        private const val CUSTOM_INPUT: Int = 1
```

Comment:
```suggestion
        private const val CUSTOM_INPUT_INDEX: Int = 1
```
CUSTOM_INPUT이라고만 한다면, 이게 입력받은 값인 것인지 키값인 것인지 잘 모를 것 같습니다.
사용하는 곳을 보니 인덱스와 관련된 것으로 보이는데, INDEX라는 단어를 넣어보면 어떨까요?

Reply:
명확한 의미 전달을 위해 INDEX를 붙여 수정하였습니다 :)